functional-tests/src/test/resources/templates/workspace/openshift/default.json

```diff
@@ -1,41 +0,0 @@
-{
```

Comment:
I'm not sure, but don't we need this file to run some basic upstream tests? (The ones which requires default workspace?)

Reply:
the file was renamed and moved to `vertx_default.json`